cypress/integration/other/prop-aliases.spec.js

```diff
@@ -9,6 +9,7 @@ const assertValues = {
   colStart: '2',
   colEnd: '3',
   gutter: '20px 25px',
+  margin: 0,
```

Comment:
This now tests `margin:0` scenario, but doesn't test positive value (i.e. `margin:10px`).
Hm, we may want to come up with another approach here, these overrides are not designed to handle multiple value assertions for a single prop alias.